rbx_dom_lua/src/EncodedValue.spec.lua

```diff
@@ -47,6 +47,36 @@ return function()
 		expect(decoded).to.equal(output)
 	end)
 
+	it("should decode a 'Faces' bit-mask value", function()
+		local input = {
+			Type = "Faces",
+			Value = 0b_111111,
```

Comment:
```suggestion
			Value = 0b111111,
```

We don't need these optional spacer characters.

Reply:
That’s fine. It was more of a personal readability choice and I wasn’t sure if there was any consensus there in the style guide.